bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -142,6 +115,56 @@ class SettingsDropdown extends Component {
     ];
   }
 
+
+  componentDidMount() {
+    // checks if the users entered fullscreen mode through the browser
+    // and if so alters the menu to have the exit fullscreen button disabled
+    window.addEventListener('resize', () => {
+      if (window.screen.width === window.innerWidth &&
+        window.screen.height === window.innerHeight) {
+        this.alterMenu(null);
+      } else {
+        this.alterMenu(this.props);
+      }
+    });
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.clicked) {
+      this.alterMenu(nextProps);
+    }
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', () => {
```

Comment:
This `removeEventListener` is not actually removing the  event added in `componentDidMount`